<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgEditClients</class>
 <widget class="QDialog" name="dlgEditClients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Client List</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-32</x>
     <y>150</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputNewClient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="clientListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btnInsert">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>27</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>27</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgEditClients</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgEditClients</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
